File: Python/testpersonalitytest.py
import personalitytest

from unittest import TestCase

class TestPersonalityTestApp(TestCase):
	def test_get_name_function_exists(self):
		self.assertIsNotNone(personalitytest.get_name())
		
	def test_get_name_function_returns_name(self):
		actual = personalitytest.get_name()
		expected = "JAMES TAURI"
		self.assertEqual(actual, expected)

File: Java/TestPersonalityTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;

public class TestPersonalityTest {
	PersonalityTest person = new PersonalityTest();
	
	@Test
	public void testGetNameFunctionExists() {
		assertNotNull(person.getName());
	}
	
	@Test	
	public void testGetNameFunctionReturnsName() {
		String actual = person.getName();
		String expected = "JAMES TAURI";
		assertEquals(actual, expected);
	}
}